app/frontend/src/features/auth/signup/CompleteSignupForm.tsx

```diff
@@ -168,13 +201,62 @@ export default function CompleteSignup() {
                 ]}
                 getOptionLabel={(option) => hostingStatusLabels[option]}
                 disableClearable
-                //below required for type inference
+                // below required for type inference
                 multiple={false}
                 freeSolo={false}
               />
             )}
           />
-          <Button onClick={completeSignup} loading={authLoading || loading}>
+          <InputLabel className={authClasses.formLabel} htmlFor="gender">
+            I identify as ....
+          </InputLabel>
+          <Controller
+            id="gender"
+            control={control}
+            name="gender"
+            defaultValue=""
+            render={({ onChange }) => (
+              <RadioGroup
+                className={classes.genderRadio}
+                aria-label="gender"
+                name="gender-radio"
+                // value={value}
+                onChange={onChange}
+              >
+                <FormControlLabel
+                  value="female"
```

Comment:
```suggestion
                  value="Woman"
```

Reply:
Since this is just displayed as is, for the moment

Reply:
This and the comments for the other gender controls still applies. Do you think it should be left as is?

Reply:
What? :D I've changed it from Woman to female as you requested, same for male & other.

Reply:
Oh! Fixed!